installer/GNUmakefile

```diff
@@ -39,11 +36,7 @@ $(info UNSIGNED_list = ${UNSIGNED_list} )
 $(info SIGNED_list   = ${SIGNED_list}   )
 $(info ALIAS_list    = ${ALIAS_list}    )
 
-target_of_ehl := tgl
-target_of_adl   := tgl
-target_of_adl-s := tgl-h
-target_of_rpl   := tgl
-target_of_rpl-s := tgl-h
+target_of_imx := imx8
```

Comment:
Could you try adding one AMD product?

Reply:
> > I think the github flows need to be updated here as well, or otherwise there will be many CI failures
> 
> @kv2019i yes, apparently PR CI is still attempting TGL XTOS builds, @lgirdwood @marc-hb do we need them? If yes, then obviously this PR cannot be merged

yep - lets make sure we update all flows. CI should not be doing any xtos builds for Intel platforms as Intel only uses Zephyr now..

Reply:
> Could you try adding one AMD product?

@marc-hb can we make it a follow-up? I'd be happy to get this right with a minimum changes